source/Octopus.Tentacle.Tests/Scripts/ScriptStateStoreTests.cs

```diff
@@ -0,0 +1,50 @@
+using System.IO;
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Tentacle.Contracts;
+using Octopus.Tentacle.Scripts;
+using Octopus.Tentacle.Tests.Support;
+using Octopus.Tentacle.Util;
+using InMemoryLog = Octopus.Tentacle.CommonTestUtils.Diagnostics.InMemoryLog;
+
+namespace Octopus.Tentacle.Tests.Scripts
+{
+    [TestFixture]
+    public class ScriptStateStoreTests
```

Comment:
These are to verify that the move in related classes doesn't result in us being unable to read these files.

Reply:
Sometimes in json serialisation we do things like include the class name and assembly name in the json.
It turns out we don't do it here, which is great to see but I wrote a test anyway i first generated the json before my changes and then we can see it passing with these changes